tests/item_pocket_test.cpp

```diff
@@ -1530,11 +1589,138 @@ TEST_CASE( "pocket favorites allow or restrict containers", "[pocket][favorite][
 // (Second argument is `avoid` item pointer, not parent item location)
 TEST_CASE( "character best pocket", "[pocket][character][best]" )
 {
-    // TODO:
-    // Includes wielded item
-    // Includes worn items
-    // Uses best of multiple worn items
-    // Skips avoided items
+    item_location loc;
+    Character &dummy = get_player_character();
+
+    // reset dummy equipment before each test.
+    reset_dummy_equipment( dummy );
+
+    WHEN( "the player is wielding a container" ) {
+        item socks( "test_socks" );
+        item container( "test_watertight_open_sealed_container_1L" );
```

Comment:
You should add itype_id definitions for all of these to the top of the file, like this:
`static const itype_id itype_test_socks( "test_socks" );`

They need to be alphabetically sorted, first by type, then by name.

Reply:
Should this be done for the other unit tests in this file as well? Or would you say that's something for a PR of its own (cleanup)?

Reply:
Depends on how much it is, but it's probably better doing it in its own PR.